PowerEditor/src/NppBigSwitch.cpp

```diff
@@ -2953,6 +2953,31 @@ LRESULT Notepad_plus::process(HWND hwnd, UINT message, WPARAM wParam, LPARAM lPa
 			return _pluginsManager.allocateIndicator(static_cast<int32_t>(wParam), reinterpret_cast<int *>(lParam));
 		}
 
+		case NPPM_GETTABCOLORID:
+		{
+			const auto view = static_cast<INT>(wParam);
+			auto tabIndex = static_cast<INT>(lParam);

```

Comment:
In general, it's better to use primitive type than macro defined type.
Here I would use `int tabIndex = static_cast<int>(lParam);` instead of `auto tabIndex = static_cast<INT>(lParam);`